### PR TITLE
Add TurbotPlugins::UrlHandler.

### PR DESCRIPTION
**Pull request by rwjblue**
Currently has special handling for github.com urls.
All other urls simply print the page title.

### CODE REVIEW

File: plugin/url_handler_cinch_plugin.rb [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Author: Robert Jackson (robert.w.jackson@me.com)
+#     Inspired by epitron/pookie: https://github.com/epitron/pookie/blob/master/handlers/url_handler.rb
```
Comment (pwelch) [inline plugin/url_handler_cinch_plugin.rb]: Could you kill the white space here?

**Comment by pwelch** [comment]
@rjackson Just a small white space issue. Everything else looks good. Did you test it in a channel?

**Comment by rwjblue** [comment]
Yep.

**Comment by rwjblue** [comment]
If you scroll back through the logs of #tambarb you can see the test output.  The Github matcher is really sweet.

Am planning on adding more matchers (i.e. RubyGems ranking and stuff for gems).

**Pull request merged**